diff --git "a/C:\\Users\\USER\\Desktop\\Personal\\ZCode\\.tmp\\z1-native-1790248539413-c0593b\\fixture-before.mjs" "b/C:\\Users\\USER\\Desktop\\Personal\\ZCode\\.tmp\\z1-native-1790248539413-c0593b\\workspace\\fixture.mjs"
index f188caa..700bad8 100644
--- "a/C:\\Users\\USER\\Desktop\\Personal\\ZCode\\.tmp\\z1-native-1790248539413-c0593b\\fixture-before.mjs"
+++ "b/C:\\Users\\USER\\Desktop\\Personal\\ZCode\\.tmp\\z1-native-1790248539413-c0593b\\workspace\\fixture.mjs"
@@ -1 +1 @@
-export const marker = 'Z1_BEFORE_7391';
+export const marker = 'Z1_AFTER_7391';
